source/classes.tex

```diff
@@ -4090,10 +4090,10 @@
 \end{example}
 
 \pnum
-A function with a deleted definition\iref{dcl.fct.def} shall
-not override a function that does not have a deleted definition. Likewise,
-a function that does not have a deleted definition shall not override a
-function with a deleted definition.%
+A deleted function\iref{dcl.fct.def} shall
+not override a function that is not a deleted function. Likewise,
+a function that is not a deleted function shall not override a
```

Comment:
If we define "deleted" as an adjective instead of defining the noun phrase "deleted function", then we can express this a bit more naturally:

```suggestion
not override a function that is not deleted. Likewise,
a function that is not deleted shall not override a
```

Reply:
Ping @zygoloid, I've updated with your suggested changes.